scripts/release_notes/commitlist.py

```diff
@@ -128,6 +128,9 @@ def categorize(commit_hash, title):
             if CommitList.keywordInFile(file, ['torch/fx', 'test_fx']):
                 category = 'fx'
                 break
+            if CommitList.keywordInFile(file, ['torch/ao', 'test/ao']):
```

Comment:
since now we `release notes: ` and `topic: ` tags, we should add a small helper to infer the tags automatically before entering this for loop, and only enter this loop if the PR didn't have the `release notes :` tag.

Reply:
I think the next PR that I added does that? https://github.com/pytorch/pytorch/pull/75020/files